dev-lang/rust-bin/rust-bin-1.25.0.ebuild

```diff
@@ -30,12 +35,29 @@ QA_PREBUILT="
 	opt/${P}/lib/rustlib/*/lib/*.rlib*
 "
 
+pkg_pretend () {
+	if [[ "$(tc-is-softfloat)" != "no" ]] && [[ ${CHOST} == armv7* ]]; then
+		die "you don't match the prebuilt rustc, sorry"
```

Comment:
Something a bit more clear wouldn't be terrible here.

Reply:
Sure, what should I write instead?

Reply:
`die "${CHOST} is not supported by upstream Rust. You must use a hard float version."`